OpenRA.Mods.D2k/Lint/CheckImportActors.cs

```diff
@@ -0,0 +1,28 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Mods.D2k.UtilityCommands;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Lint
+{
+	public class CheckImportActors : ILintRulesPass
+	{
+		public void Run(Action<string> emitError, Action<string> emitWarning, Ruleset rules)
+		{
+			foreach (var actorData in D2kMapImporter.ActorDataByActorCode.Values)
+			{
+				if (!rules.Actors.Keys.Contains(actorData.First))
```

Comment:
Yes - the first is doing the lookup using the dictionary, the second requires a linear search.